docs/Installation.md

```diff
@@ -0,0 +1,194 @@
+Zonemaster::Engine installation] and [Zonemaster::Backend][Zonemaster::Backend
+installation].
+
+> **Note:** [Zonemaster::Backend], and consequently [Zonemaster::Engine] and
+> [Zonemaster::LDNS] are dependencies of Zonemaster::GUI. Zonemaster::LDNS has a
+> special installation requirement, and both Zonemaster::Engine and
+> Zonemaster::Backend have lists of dependencies that you may prefer to
+> install from your operating system distribution (rather than CPAN).
+> We recommend following the Zonemaster::Engine and Zonemaster::Backend
+> installation instruction.
+
+For details on supported versions of Perl and operating system for
+Zonemaster::CLI, see the [declaration of prerequisites].
+
+
+## Installation
+
+This instruction covers the following operating systems:
+
+ 1. [CentOS](#1-centos)
+ 2. [Debian](#2-debian)
+ 3. [Docker](#3-docker)
+ 4. [FreeBSD](#4-freebsd)
+ 5. [Ubuntu](#5-ubuntu)
+
+
+### 1. CentOS
+
+1) Install additional prerequisite packages.
+
+    sudo cpan -i Dancer Text::Markdown Template JSON
+
+2) Get the source code.
+
+    git clone https://github.com/dotse/zonemaster-gui.git
+
+3) Change to the source code directory.
+
+    cd zonemaster-gui
+
+4) Install the Perl modules.
+
+    perl Makefile.PL
+    make
+    make test
+    make install
+
+5) Create a directory for the webapp parts, and copy them there.
+
+    mkdir -p /usr/local/share/zonemaster
+    cp -a zm_app /usr/local/share/zonemaster
+
+6) Start the server:
+
+    starman --listen=:80 --daemonize /usr/local/share/zonemaster/zm_app/bin/app.pl
+
+
+### 2. Debian
+
+1) Install added prerequisite packages:
+
+    sudo apt-get install libdancer-perl libtext-markdown-perl libtemplate-perl libjson-any-perl
+
+2) Get the source code.
+
+    git clone https://github.com/dotse/zonemaster-gui.git
+
+3) Change to the source code directory.
+
+    cd zonemaster-gui
+
+4) Install the Perl modules.
+
+    perl Makefile.PL
+    make
+    make test
+    sudo make install
+
+5) Create a directory for the webapp parts, and copy them there.
+
+    sudo mkdir -p /usr/share/doc/zonemaster
+    sudo cp -a zm_app /usr/share/doc/zonemaster
+
+6) Start the server:
+
+    sudo starman --listen=:80 /usr/share/doc/zonemaster/zm_app/bin/app.pl
+
+The Doc directory in the source code also has an example Upstart file for the Web GUI starman server.
+
+
+### 3. Docker
+
+Install the docker package on your OS
+
+Follow the installation instructions for your OS -> https://docs.docker.com/engine/installation/linux/
+	
+Pull the docker image containing the complete Zonemaster distribution (GUI + Backend + Engine)
+
+	docker pull afniclabs/zonemaster-gui
+
+Start the container in the background
+
+	docker run -t -p 50080:50080 afniclabs/zonemaster-gui
+	
+Use the Zonemaster GUI by pointing your browser at
+
+	http://localhost:50080/
+	
+Use the Zonemaster engine from the command line
+
+	docker run -t -i afniclabs/zonemaster-gui bash
+	

```

Comment:
Update with cpan installation.

Reply:
Can you provide a wording that I can use?